android-sdk/android/src/test/kotlin/io/logto/sdk/android/auth/social/alipay/AlipaySocialSessionTest.kt

```diff
@@ -0,0 +1,145 @@
+package io.logto.sdk.android.auth.social.alipay
+
+import android.os.Bundle
+import com.alipay.sdk.app.OpenAuthTask
+import com.google.common.truth.Truth.assertThat
+import io.logto.sdk.android.auth.logto.LogtoWebViewAuthActivity
+import io.logto.sdk.android.auth.social.SocialException
+import io.logto.sdk.android.completion.Completion
+import io.mockk.Runs
+import io.mockk.clearAllMocks
+import io.mockk.every
+import io.mockk.just
+import io.mockk.mockk
+import io.mockk.mockkConstructor
+import io.mockk.verify
+import org.junit.After
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.Robolectric
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
+class AlipaySocialSessionTest {
+    private val activityController = Robolectric.buildActivity(LogtoWebViewAuthActivity::class.java)
+
+    private val activity = activityController.get()
+    private val mockCompletion: Completion<SocialException, String> = mockk()
+
+    private val socialExceptionCapture = mutableListOf<SocialException?>()
+    private val continueSignInUriCapture = mutableListOf<String?>()
+
+    @After
+    fun tearDown() {
+        clearAllMocks()
+    }
+
+    @Test
+    fun start() {
```

Comment:
this name should align with other cases? also in other PRs

Reply:
Maybe refactor together.
LOG-2269